janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -332,6 +335,8 @@ public void mutate(Map<String, Map<String, IndexMutation>> mutations, KeyInforma
                 String collectionName = stores.getKey();
                 String keyIdField = getKeyFieldId(collectionName);
 
+                Map<String, String> fieldDeletes = new HashMap<String, String>(1) {{ put("set", null); }};
```

Comment:
Can't use ImmutableMap here since the value is null.  Will use Collections.unmodifiableMap here and ImmutableMap elsewhere.